translations/en-us.yml

```diff
@@ -598,7 +598,11 @@ node_card:
             withdrawn: Withdrawn
             archiving: Archiving
     tags: 'Tags:'
+    schema: 'Schema:'
+    contributors: 'Contributors:'
+    description: 'Description:'
```

Comment:
With the pseudo element `::after{ content: ": "; };` on `.Labels` no need for ":" here.